dev-python/nbconvert/nbconvert-9999.ebuild

```diff
@@ -22,9 +22,10 @@ fi
 
 LICENSE="BSD"
 SLOT="0"
-IUSE="test"
+IUSE="doc test"
 
 RDEPEND="
+	>=app-text/pandoc-1.12.1
```

Comment:
For ipython (https://github.com/gentoo/gentoo/blob/master/dev-python/ipython/ipython-3.2.1.ebuild#L144) pandoc was optional. As far as I get it nbconvert is an alternative to pandoc, based on jinja2 templates.

I would like to avoid adding pandoc to the dependencies as this would pull in 90 new packages, mostly from `dev-haskell`. Please recheck, as i can't find any word on pandoc in the nbconvert repository. Further, tests pass without pandoc installed.